test-cases/javascript/interactive/multi-step-request-event-attribute.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<body id="thebody">
+    <title>CrawlMaze - Testbed for Web Crawlers - form tag</title>
+    <h1> multi-step GET method </h1>
+    <input type="button" onClick="showSecretButton();" value='show secret button' id='but'/>
+    <input type="button" onClick="doNothing();" value='decoy button' id='decoy'/>
```

Comment:
is the value and id needed here?

Reply:
The id was not used, thanks for the tip! Is there some way other than specifying a value that adds text to an input with the button type, or could you elaborate on what exactly you would change about the value?